test.txt

```diff
@@ -1,5 +1,2 @@
 This is a test file.
 This is a test file.
-
-My report work is completed.
```

Comment:
This code is not correct. please fix it.

Reply:
I have correct the code.